doc-changelog/action.yml

```diff
@@ -156,14 +165,12 @@ runs:
     - name: "Get labels in the pull request"
       if: ${{ inputs.use-conventional-commits == 'false' }}
       env:
-        OWNER: ${{ github.repository_owner }}
-        REPO_NAME: ${{ github.event.repository.name }}
-        PR_NUMBER: ${{ github.event.number }}
         GH_TOKEN: ${{ inputs.token }}
+        REPO_NAME: ${{ github.event.repository.name }}
       shell: bash
       run: |
         # Get the labels in the pull request
-        pr_labels=$(gh api repos/${{ env.OWNER }}/${{ env.REPO_NAME }}/pulls/${{ env.PR_NUMBER }} --jq '.labels.[].name')
+        pr_labels=$(gh api repos/${{ github.repository_owner }}/${{ env.REPO_NAME }}/pulls/${{ github.event.number }} --jq '.labels.[].name')
```

Comment:
Why are these two not changed according to the guidelines? We should revert the github context entries to env vars, right?

Reply:
I didn't change the repository_owner & event.number because the GitHub docs said the contexts that could have untrusted input typically end with `body`, `default_branch`, `email`, `head_ref`, `label`, `message`, `name`, `page_name`, `ref`, and `title`. I think the repository owner and PR number would probably be fine. Hopefully GitHub has some naming rules for organizations/usernames

Reply:
That's fine - approving!